package.json

```diff
@@ -35,15 +35,15 @@
     "scripts": {
         "clean": "rimraf lib *.jar *.log",
         "compile": "tsc",
-        "test": "node scripts/download-and-run-rc.js && mocha 'test/**/*.js'",
-        "test:unit": "mocha 'test/unit/**/*.js'",
-        "test:integration": "node scripts/download-and-run-rc.js && mocha 'test/integration/**/*.js'",
+        "test": "node scripts/test-runner.js all",
+        "test:unit": "node scripts/test-runner.js unit",
+        "test:integration": "node scripts/test-runner.js integration",
         "validate-user-code": "tsc --build test/user_code/tsconfig.json",
-        "precoverage": "node scripts/download-and-run-rc.js",
-        "coverage": "rimraf coverage && nyc node_modules/mocha/bin/_mocha 'test/**/*.js' -- --reporter-options mochaFile=report.xml --reporter mocha-junit-reporter",
+        "coverage": "node scripts/test-runner.js coverage",
         "pregenerate-docs": "rimraf docs",
         "generate-docs": "typedoc --options typedoc.json",
-        "lint": "eslint --ext .ts src && eslint --plugin mocha test && eslint code_samples"
+        "lint": "eslint --ext .ts src && eslint --plugin mocha test && eslint code_samples",
+        "startrc": "node scripts/test-runner.js startrc"
```

Comment:
We could also think adding a downloadrc script here

Reply:
I'd postpone this enhancement as we don't need it now.